2024-04-04 21:46:48,658 - ERROR - Hubo un error al recuperar los datos la sintaxis de entrada no es válida para tipo integer: «None»
LINE 1: ...on from usuarios_sesiones a where a.cod_usuario = 'None' and...
                                                             ^

2024-04-04 21:49:02,235 - ERROR - Hubo un error al recuperar los datos la sintaxis de entrada no es válida para tipo integer: «None»
LINE 1: ...on from usuarios_sesiones a where a.cod_usuario = 'None' and...
                                                             ^

2024-04-04 21:58:21,621 - ERROR - Hubo un error al recuperar los datos name 'psycopg2' is not defined
2024-04-04 21:58:28,797 - ERROR - Hubo un error al recuperar los datos name 'psycopg2' is not defined
2024-04-04 21:59:30,863 - ERROR - Hubo un error al recuperar los datos name 'psycopg2' is not defined
2024-04-04 22:00:33,696 - ERROR - Hubo un error al recuperar los datos name 'psycopg2' is not defined
2024-04-04 22:00:45,028 - ERROR - Hubo un error al recuperar los datos name 'psycopg2' is not defined
2024-04-04 22:01:35,096 - ERROR - Hubo un error al recuperar los datos la sintaxis de entrada no es válida para tipo integer: «None»
LINE 1: ...on from usuarios_sesiones a where a.cod_usuario = 'None' and...
                                                             ^

2024-04-04 22:02:50,123 - ERROR - Hubo un error al recuperar los datos name 'psycopg2' is not defined
2024-04-04 22:03:18,892 - ERROR - Hubo un error al recuperar los datos name 'psycopg2' is not defined
2024-04-04 22:04:25,607 - ERROR - Hubo un error al recuperar los datos name 'psycopg2' is not defined
2024-04-04 22:04:33,012 - ERROR - Hubo un error al recuperar los datos name 'psycopg2' is not defined
2024-04-04 22:05:05,391 - ERROR - Hubo un error al recuperar los datos can only concatenate str (not "list") to str
2024-04-04 22:05:15,181 - ERROR - Hubo un error al recuperar los datos can only concatenate str (not "list") to str
2024-04-04 22:07:41,964 - ERROR - Hubo un error al recuperar los datos list indices must be integers or slices, not str
2024-04-04 22:07:47,071 - ERROR - Hubo un error al recuperar los datos list indices must be integers or slices, not str
2024-04-04 22:08:15,681 - ERROR - Hubo un error al recuperar los datos can only concatenate str (not "tuple") to str
2024-04-04 22:26:36,344 - INFO - Resultado: [(16,)]
2024-04-04 22:27:19,905 - INFO - Resultado: [(16,)]
2024-04-04 22:27:44,100 - ERROR - Hubo un error al recuperar los datos name 'psycopg2' is not defined
2024-04-04 22:29:45,414 - ERROR - Hubo un error al recuperar los datos __class__ assignment: 'RealDictCursor' object layout differs from 'CursorDebugWrapper'
2024-04-04 22:47:58,799 - INFO - Resultado: [(16,)]
2024-04-04 22:47:58,824 - ERROR - Hubo un error al recuperar los datos la sintaxis de entrada no es válida para tipo integer: «None»
LINE 1: ...on from usuarios_sesiones a where a.cod_usuario = 'None' and...
                                                             ^

2024-04-04 22:49:56,264 - INFO - Resultado: [(16,)]
2024-04-04 22:49:56,273 - ERROR - Hubo un error al recuperar los datos la sintaxis de entrada no es válida para tipo integer: «None»
LINE 1: ...on from usuarios_sesiones a where a.cod_usuario = 'None' and...
                                                             ^

2024-04-04 22:51:04,526 - INFO - Resultado: (16,)
2024-04-04 22:51:04,529 - ERROR - Hubo un error al recuperar los datos la sintaxis de entrada no es válida para tipo integer: «None»
LINE 1: ...on from usuarios_sesiones a where a.cod_usuario = 'None' and...
                                                             ^

2024-04-04 22:51:48,393 - INFO - Resultado: (16,)
2024-04-04 22:51:48,401 - INFO - Resultado: None
2024-06-10 12:39:01,001 - ERROR - Hubo un error al recuperar los datos no existe la relación «usuarios_espacios»
LINE 2:                     from usuarios_espacios ue, espacios es  
                                 ^

2024-06-10 12:45:03,781 - ERROR - Hubo un error al recuperar los datos no existe la relación «usuarios_espacios»
LINE 2:                     from usuarios_espacios ue, espacios es  
                                 ^

2024-06-10 12:48:52,796 - ERROR - Hubo un error al recuperar los datos no existe la relación «usuarios_espacios»
LINE 2:                     from usuarios_espacios ue, espacios es  
                                 ^

2024-06-10 12:49:41,220 - ERROR - Hubo un error al recuperar los datos no existe la relación «usuarios_espacios»
LINE 2:                     from usuarios_espacios ue, espacios es  
                                 ^

2024-06-10 12:49:52,723 - ERROR - Hubo un error al recuperar los datos no existe la relación «usuarios_espacios»
LINE 2:                     from usuarios_espacios ue, espacios es  
                                 ^

2024-06-10 12:50:01,044 - ERROR - Hubo un error al recuperar los datos no existe la relación «usuarios_espacios»
LINE 2:                     from usuarios_espacios ue, espacios es  
                                 ^

2024-06-10 12:53:23,745 - INFO - Se obtuvieron 1 espacios de trabajo.
2024-06-10 12:53:41,357 - INFO - Se obtuvieron 2 espacios de trabajo.
2024-06-10 12:56:46,263 - ERROR - Hubo un error al recuperar los datos error de sintaxis en o cerca de «ue»
LINE 4:                     ue.cod_usuario = '16'
                            ^

2024-06-10 12:57:28,781 - INFO - Se obtuvieron 2 espacios de trabajo.
2024-06-10 14:12:22,566 - INFO - Se obtuvieron 2 espacios de trabajo.
2024-06-10 14:13:00,588 - INFO - Se obtuvieron 2 tableros para el espacio de trabajo: 5
2024-06-10 14:13:37,985 - INFO - Se obtuvieron 2 tableros para el espacio de trabajo: 5
2024-06-28 00:20:05,241 - INFO - Se obtuvieron 2 espacios de trabajo.
